common/src/main/java/dev/isxander/yacl3/api/controller/DropdownStringControllerBuilder.java

```diff
@@ -0,0 +1,15 @@
+package dev.isxander.yacl3.api.controller;
+
+import dev.isxander.yacl3.api.Option;
+import dev.isxander.yacl3.impl.controller.DropdownStringControllerBuilderImpl;
+
+import java.util.List;
+
+public interface DropdownStringControllerBuilder<T> extends StringControllerBuilder {
+	DropdownStringControllerBuilderImpl allowedValues(List<String> allowedValues);
+	DropdownStringControllerBuilderImpl allowedValues(String[] allowedValues);
```

Comment:
I wonder if there could also be a distinction between 'allowed' and 'suggested'. One being mandatory, one being not?

Reply:
In my first implementation, I had that. But then I realised that it works as suggested values if the library user simply adds the empty string to the allowed values list.

Reply:
This is very unclear. You should add the builder method to suggest only that internally adds this to `allowedValues`. Make sure that calling it twice doesn't add it twice.